chihacknight/decarbonize-my-state
# Add Power Plant Detail Pages

## Overview

**Note:** PR description by @vkoves who took over after the initial date import work.

Creates power plant detail pages, linked out on the state details pages, which show the basic stats we have about power plants, basic calculations to frame some of that data in more understandable terms, and a link to view the power plant in Google Maps (via the plant's lat long coordinates).

| Desktop | Mobile |
| --- | --- |
| ![Screenshot from 2023-04-23 15-26-32](https://user-images.githubusercontent.com/3187531/233864685-1d2f9089-c603-4833-b260-0871e45f5923.png) |  ![Screenshot from 2023-04-23 15-28-06](https://user-images.githubusercontent.com/3187531/233864684-b168e642-bd51-4492-9464-e9708be886dd.png) |

<details>
 <summary>State Details Page Screenshot</summary>

  
![Screenshot from 2023-04-23 15-30-51](https://user-images.githubusercontent.com/3187531/233864595-90cddc88-8fe4-4bf5-b43a-f078a9db0ba3.png)

</details

Closes #188 and #190

Implements designs from #184

## Testing Instructions

- Confirm that you can view power plants on multiple states and see accurate data on desktop and mobile
